src/webhook.ts

```diff
@@ -12,8 +13,11 @@ if (
 
 serve((req: Request) => {
   if (req.url.endsWith("/trigger")) {
-    run();
-    return Response.json({ message: "Triggered backport" });
+    backport.run();
+    labels.run();
```

Comment:
We should only run this step if the previous one was successful…

Reply:
These are both async and don't depend on one another, we don't wait for them to complete before acknowledging the webhook

Reply:
That's the correct thing to do, though. 
Even if the PR can't be backported, it was still merged, in which case these specific labels are no longer needed.

Reply:
These functions share no state at all. The first creates backports for merged PRs, the second removes labels from merged PRs. Both do a global search on each run, they don't work only on the push that triggered the webhook